Markdown/Markdown.sublime-syntax

```diff
@@ -2956,6 +2960,172 @@ contexts:
+    - meta_include_prototype: false
+    - match: \$\$
+      scope: punctuation.definition.math.begin.latex
+      set:
+        - list-math-inline-block-end
+        - list-math-block-content
+
+  list-math-inline-block-end:
+    - meta_include_prototype: false
+    - meta_scope:
+        markup.math.block.markdown
+        text.tex.latex.embedded.markdown
+        meta.environment.math.block.dollar.latex
+    - match: \$\$
+      scope: punctuation.definition.math.end.latex
+      pop: 1
+    - match: ''
+      fail: list-math-blocks
+
+  list-math-block-content:
+    - meta_include_prototype: false
+    - match: ''
+      embed: math-content
+      escape: (?=\$\$)|{{list_paragraph_end}}|{{list_setext_heading_escape}}
+      pop: 1 # embed math once
+
+  math-blocks:
+    - match: (?=(?:^[ ]{,3})?\$\$)
+      branch_point: math-blocks
+      branch:
+        - math-block-begin
+        - math-inline-block-begin
+
+  math-block-begin:
+    # Leading and trailing whitespace of standalone math blocks are scoped
+    # `markup.math` in case background color is assigned.
+    - meta_include_prototype: false
+    - match: ^[ ]{,3}(\$\$)
+      captures:
+        1: text.tex.latex.embedded.markdown
+           meta.environment.math.block.dollar.latex
+           punctuation.definition.math.begin.latex
+      set:
+        - math-block-end
+        - math-block-content
+    - match: ''
+      fail: math-blocks
+
+  math-block-end:
+    - meta_include_prototype: false
+    - meta_scope: markup.math.block.markdown
+    - meta_content_scope:
+        text.tex.latex.embedded.markdown
+        meta.environment.math.block.dollar.latex
+    - match: (\$\$)\s*$\n?
+      captures:
+        1: text.tex.latex.embedded.markdown
+           meta.environment.math.block.dollar.latex
+           punctuation.definition.math.end.latex
+      pop: 1
+    - match: ''
+      fail: math-blocks
+
+  math-inline-block-begin:
+    # Math blocks preceeded or followed by normal text are treated as inline raw
+    # code blocks, scoped `markup.math` excluding leading or trailing whitespace.
+    - meta_include_prototype: false
+    - match: \$\$
+      scope: punctuation.definition.math.begin.latex
+      set:
+        - math-inline-blocks-end
+        - math-block-content
+
+  math-inline-blocks-end:
+    - meta_include_prototype: false
+    - meta_scope:
+        markup.math.block.markdown
+        text.tex.latex.embedded.markdown
+        meta.environment.math.block.dollar.latex
+    - match: \$\$
+      scope: punctuation.definition.math.end.latex
+      pop: 1
+    - match: ''
+      fail: math-blocks
+
+  math-block-content:
+    - meta_include_prototype: false
+    - match: ''
+      embed: math-content
+      escape: (?=\$\$)|{{paragraph_end}}|{{setext_heading_escape}}
+      pop: 1 # embed math once
+
+  math-inline:
+    # Consume any remaining $$ to prevent deadlock
+    # (workaround for sublimehq/sublime_text/issues/5415)
```

Comment:
if you put the full URL, we can just right-click & open URL in browser from within Sublime